### DISCUSSION TITLE
过期：怎么给"mpv-osc-tethys"主题的进度条添加缩略图。

### DISCUSSION THREAD

**Discussion by lyturn** [hooke007/mpv_PlayKit#108] [回收]
我用deus0ww 的 Thumbnailer.lua Thumbnailer_Worker.lua脚本测试了一下，它不显示缩略图。
![015](https://user-images.githubusercontent.com/91406367/161412600-f85b76fd-b5f9-4413-9393-1262c520622a.png)

**Comment by hooke007** [comment]
不显示是正常的。
绘制缩略图和通信Thumbnailer脚本的代码都是写在osc脚本里边的

**Comment by lyturn** [comment]
osc脚本怎么修改

**Reply by hooke007** [reply]
自行参考 thumbnailer_osc

**Comment by lyturn** [comment]
是参考这个文件吗具体哪一处 Thumbnailer_OSC.lua

**Reply by hooke007** [reply]
自然是所有和原始osc不一样的地方
我的注释里已经标明了原始脚本
https://github.com/hooke007/MPV_lazy/blob/94637dcbf0fd317a20bee4b9dbe1caef6e34d441/portable_config/scripts/osc_lazy.lua#L2-L5
